doc/rst/high_bandwidth_endpoints.rst

```diff
@@ -0,0 +1,146 @@
+|newpage|
+
+************************************
+High bandwidth isochronous endpoints
+************************************
+
+``lib_xud`` provides support for high bandwidth isochronous endpoints on high-speed USB devices, as described in the
+Section 5.9.2 of the `USB 2.0 specification <https://usb.org/document-library/usb-20-specification>`_.
+This enables higher data throughput by allowing multiple transactions per microframe.
+
+The current implementation in ``lib_xud`` has the following limitations:
+
+* A maximum of 2 transactions per microframe is supported (the USB specification allows up to 3).
+* The maximum data payload size per transaction is the same across all high bandwidth endpoints
+* The maximum data payload size per transaction must be a multiple of 4 bytes.
+* This feature is supported only on `xcore.ai` series devices running at an increased system frequency of 800 MHz due to the performance demands of high bandwidth transfers.
+
+Enabling high bandwidth ISO endpoint support
+============================================
+
+High bandwidth ISO endpoint support is disabled by default. To enable it, the application must define the
+``XUD_USB_ISO_MAX_TXNS_PER_MICROFRAME`` macro as ``2``. This allows up to two transactions per microframe.
+By default, this macro is set to ``1`` in ``xud.h``, limiting operation to a single transaction and disabling
+high bandwidth transfers. Applications can override the default by re-defining the macro in a custom ``xud_conf.h`` file.
```

Comment:
again, I think the mechanism for actually setting this should be covered elsewhere in the doc.